addon/models/new-platform-flexberry-g-i-s-map-layer.js

```diff
@@ -25,7 +25,12 @@ let Model = Projection.Model.extend(NewPlatformFlexberyGISMapLayerModelMixin, Le
     let stringToDeserialize = this.get('settings');
     if (!Ember.isBlank(stringToDeserialize)) {
       try {
-        return JSON.parse(stringToDeserialize);
+        let result = JSON.parse(stringToDeserialize);
+        if (!Ember.get(result, 'opacity')) {
+          Ember.set(result, 'opacity', 1);
+        }
```

Comment:
I agree. 1 must be set into layer.settings.opacity as default value in "onMapLayerAdd" action handler and in base-layer's setOpacity method. This logic is unnecessary here.